### ISSUE TITLE
Feature: add misspell as default

### ISSUE THREAD

**Issue by eduncan911** [alecthomas/gometalinter#118]
Could you add `misspell` as one of the default includes?  

`vim-go`'s `go_metalinter_command` doesn't seem to be easily extendable to add the parameter.  setting it seems to override everything else and disables the autosave stuff. 

https://github.com/client9/misspell

```
$ gometalinter --linter='misspell:misspell ./*.go:PATH:LINE:COL:MESSAGE' .
bdd.go:60:24:warning: found "implemention" a misspelling of "implementation" (misspell)
doc.go:205:27:warning: found "avaliable." a misspelling of "available." (misspell)
```

**Comment by alecthomas** [comment]
Seems reasonable, but you might also want to file a bug with vim-go if this is not  possible there.

**Comment by client9** [comment]
Hi, I've been delaying adding PR for client9/misspell until the false-positive rate met my standard.  I think it's solid now.

**Comment by client9** [comment]
PR is in!  

cc @shawnps
